editor/assets/chunks/builtin/uniforms/cc-reflection-probe.chunk

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2017-2020 Xiamen Yaji Software Co., Ltd.
+
+#pragma define-meta USE_REFLECTION_PROBE range([0, 2])
+#pragma define NONE 0
```

Comment:
global macro should be named with usage, such as REFLECTION_PROBE_NONE / REFLECTION_PROBE_CUBE / REFLECTION_PROBE_PLANAR